src/braintree.android.ts

```diff
@@ -37,16 +37,12 @@ export class Braintree extends Observable {
 
         let clientTokenMethod = dropInRequest.getClass().getMethod("clientToken", [java.lang.String.class]);
 
-        let isAndroidPayEnabled = dropInRequest.getClass().getMethod("isAndroidPayEnabled", []);
-        let booleanType = isAndroidPayEnabled.getReturnType();
-
         if (options.collectDeviceData) {
-            let collectDeviceDataMethod = dropInRequest.getClass().getMethod("collectDeviceData", [booleanType]);
-            collectDeviceDataMethod.invoke(dropInRequest, [true]);
```

Comment:
Before NativeScript Android runtime had bug with metadata generation that's why I have used reflection. You can get more details from here: https://github.com/NativeScript/android-runtime/issues/831#issuecomment-325651230
So, now it's working without reflection?

Reply:
Yeah, I found the issue with the runtime bug. It seems to work without the reflection now, yes! I've tested everything, but would appreciate if you'd double check ;)